app/http.php

```diff
@@ -539,19 +549,23 @@ function removeAllRuntimes(Table $activeRuntimes, Pool $orchestrationPool): void
     ->desc('Create an execution')
     // Execution-related
     ->param('runtimeId', '', new Text(64), 'The runtimeID to execute.')
-    ->param('payload', '', new Text(8192), 'Data to be forwarded to the function, this is user specified.', true)
-    ->param('variables', [], new Assoc(), 'Environment variables passed into runtime.', true)
+    ->param('body', '', new Text(20971520), 'Data to be forwarded to the function, this is user specified.', true)
```

Comment:
on what basis is this limit set? what if the payload exceeds this size? For example when uploading a file in chunks ? Can we make sure it accepts 5MB ?

Reply:
This should support up to 20MB, which is limit that we consider reasonable even in Appwrite.
5MB chunk uploads should work just fine. Trying to upload 30MB file at once will fail.